spec/controllers/admin/users_controller_spec.rb

```diff
@@ -3,10 +3,35 @@
 describe Admin::UsersController do
 
   describe 'GET index' do
-    subject { get :index }
+    subject(:get_index) { get :index }
 
     authenticated_as(:admin) do
       it { should be_success }
+
+      describe "populating @users" do
+        subject(:users) { get_index; assigns(:users) }
+
+        it { should include(FactoryGirl.create(:user, :member)) }
+        it { should_not include(FactoryGirl.create(:user, :admin)) }
+      end
+    end
+
+    it_behaves_like "action requiring authentication"
+    it_behaves_like "action authorizes roles", [:admin]
+  end
+
+  describe 'GET index_admins' do
+    subject(:get_index) { get :index_admins }
+
+    authenticated_as(:admin) do
+      it { should be_success }
+
+      describe "populating @users" do
+        subject(:users) { get_index; assigns(:users) }
+
+        it { should include(FactoryGirl.create(:user, :admin)) }
+        it { should_not include(FactoryGirl.create(:user, :member)) }
```

Comment:
OOC: as a standard, we are using RSpec 2 `should` syntax instead of `is_expected`?

I'm fine with that, just wanna be sure of which syntax I should use next time I spin up an application.